pyserver/main.py

```diff
@@ -343,24 +532,34 @@ def sort_claims_tree(claims_tree:ClaimTree, sort_type : str = "deduplicate", log
       "u/4/N_tok" : TK_TOT,
       "u/4/in_tok": TK_IN,
       "u/4/out_tok" : TK_OUT,
-      "dedup_subclaims" : wandb.Table(data=dupe_logs, columns = ["sub_claim_list", "deduped_claims"]),
+      "deduped_claims" : wandb.Table(data=dupe_logs, columns = ["full_flat_claims", "deduped_claims"]),
       "t3c_report" : wandb.Table(data=report_data, columns = ["t3c_report"])
-      # TODO: do we need this?
-      #"num_claims" : NUM_CLAIMS,
-      #"num_topics_post_sort" : NUM_TOPICS_STEP_3
     })
-  return {"tree" : full_sort_tree, "per_topic_dupes" : nested_claims, "dupe_counts" : dupe_counts}
+  net_usage = {"total_tokens" : TK_TOT,
+               "prompt_tokens" : TK_IN,
+               "completion_tokens" : TK_OUT}
+
+  return {"tree" : full_sort_tree, "usage" : net_usage} 
+# TODO: RETURN USAGE DUH
 
 #TODO: refactor into separate testing script
 
 ######################
```

Comment:
I think testing should be moved into a separate file.

Reply:
yes absolutely, there's a todo for that right below, and there's a pytest set up with test_pipeline.py—started making those tests work, realized it's all around equality assertions and we primarily want to test LLM calls that may not be entirely deterministic—will keep iterating